pkg/executables/sonobuoy.go

```diff
@@ -31,19 +31,23 @@ func (k *Sonobuoy) Run(ctx context.Context, contextName string, args ...string)
 		"--wait",
 	}
 	executionArgs = append(executionArgs, args...)
-	_, err := k.executable.Execute(ctx, executionArgs...)
+	output, err := k.executable.Execute(ctx, executionArgs...)
+	command := strings.Join(executionArgs, " ") + "\n"
 	if err != nil {
-		return "", fmt.Errorf("error executing sonobuoy: %v", err)
+		return command, fmt.Errorf("error executing sonobuoy: %v", err)
 	}
+	return command + output.String(), err
```

Comment:
should we add a delimiter between command and output.String(), maybe a new line?

Reply:
Above `command := strings.Join(executionArgs, " ") + "\n"`

Reply:
oh I see nvm then